addons/product/models/product_template.py

```diff
@@ -5,7 +5,7 @@
 import logging
 from collections import defaultdict
 
-from odoo import api, fields, models, tools, _, SUPERUSER_ID
+from odoo import Command, SUPERUSER_ID, _, api, fields, models, tools
```

Comment:
Command import is not used anymore, you can drop this part of the diff (same for the test file)